### ISSUE TITLE
Community moderation

### ISSUE THREAD

**Issue by JediMaster25** [LemmyNet/lemmy#2619]
I think it's time to concede that the moderation system isn't working. Many communities don't have moderation because the moderators haven't connected to the platform in months and people still don't use /c/community_requests. People probably don't want to moderate because it's like a full time job that should be paid. I think [a trust level system like the one from discourse](https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/) would be more effective, that way all the responsibility doesn't fall on a few users that will get tired fast. The only action the admins should need to take is changing the trust levels of the top of the hierarchy to shape their instance the way they want it to be. Other than that they shouldn't need to do anything. I think a system that had an increasingly small number of users on the top trust levels so that admins only have to moderate a few users would be the best.

How to implement community moderation on lemmy so that there are no moderators needed?

> There are a few potential ways to implement community moderation on Lemmy:
> 1. Upvoting and downvoting system: Allow users to upvote or downvote content, and have a threshold for visibility or removal. This allows the community to collectively decide which content is appropriate and valuable.
> 2. User reporting system: Allow users to report content that they believe is inappropriate or violates community guidelines. This can be reviewed by a group of trusted community members who can decide on a course of action.
> 3. Implement a reputation system: This allows users to earn points or reputation for their contributions to the community. This can be based on the number of upvotes their posts receive, the quality of their comments, and the amount of time they spend engaging with the community. Users with higher reputations can have more privileges, such as being able to report inappropriate content or flag inappropriate comments.
> 4. Use AI moderation: Artificial intelligence can be used to automatically identify and remove inappropriate content. This can be done by training the AI on a large dataset of content and comments, and then using it to identify patterns and trends that are indicative of inappropriate content.
> 5. Use a combination of AI and human moderation: While AI can be effective at identifying and removing inappropriate content, it can still miss some things. By combining AI moderation with human oversight, you can ensure that the content on your platform is as clean and appropriate as possible.
> 6. Self-moderation by users: Encourage users to use their own discretion and judgment when posting content, and to report or flag content that they believe is inappropriate or violates community guidelines.
> 7. Use automated filters and moderation tools: Implement tools such as language filters or content moderation algorithms to automatically remove inappropriate content.
> 8. Utilize community-driven rules: By allowing users to create and vote on their own rules and guidelines for each community, the users can take ownership of its own moderation process. This can also help to ensure that the rules are fair and relevant to the community.
> 
> Overall, it is important to establish clear guidelines and expectations for the community, and to empower users to participate in the moderation process. By fostering a sense of ownership and responsibility among the community, it may be possible to reduce the need for dedicated moderators.
> 
> -- ChatGPT

List the problems with Lemmy moderation.

> 1. Bias: Moderators on Lemmy often have their own biases and agendas, which can lead to unfair moderation decisions.
> 2. Lack of transparency: Lemmy moderators often operate behind the scenes, and their decision-making process is not always clear to the community.
> 3. Lack of accountability: Lemmy moderators are not held accountable for their actions, which can lead to abuse of power.
> 4. Lack of consistency: Lemmy moderators may apply different standards to different users, leading to inconsistency in moderation decisions.
> 5. Lack of training: Some Lemmy moderators may not have the necessary skills or knowledge to effectively moderate a community.
> 6. Limited resources: Lemmy moderators often have limited resources and time, which can lead to a backlog of moderation tasks and a slow response to community issues.
> 7. Community polarization: Lemmy moderators may inadvertently contribute to community polarization by moderating content in a way that reinforces existing divisions.
> 8. Censorship: Lemmy moderators may remove content that is controversial or unpopular, leading to accusations of censorship.
> 9. Conflicts of interest: Lemmy moderators may have conflicts of interest, such as a financial stake in the sublemmy they are moderating, which can influence their moderation decisions.
> 10. Lack of diversity: The Lemmy moderation team may not be representative of the community it serves, leading to a lack of diverse perspectives and viewpoints.    Bias: Moderators on Lemmy often have their own biases and agendas, which can lead to unfair moderation decisions.
> 11. Lack of transparency: Lemmy moderators often operate behind the scenes, and their decision-making process is not always clear to the community.
> 12. Lack of accountability: Lemmy moderators are not held accountable for their actions, which can lead to abuse of power.
> 13. Lack of consistency: Lemmy moderators may apply different standards to different users, leading to inconsistency in moderation decisions.
> 14. Lack of training: Some Lemmy moderators may not have the necessary skills or knowledge to effectively moderate a community.
> 15. Limited resources: Lemmy moderators often have limited resources and time, which can lead to a backlog of moderation tasks and a slow response to community issues.
> 16. Community polarization: Lemmy moderators may inadvertently contribute to community polarization by moderating content in a way that reinforces existing divisions.
> 17. Censorship: Lemmy moderators may remove content that is controversial or unpopular, leading to accusations of censorship.
> 18. Conflicts of interest: Lemmy moderators may have conflicts of interest, such as a financial stake in the sublemmy they are moderating, which can influence their moderation decisions.
> 19. Lack of diversity: The Lemmy moderation team may not be representative of the community it serves, leading to a lack of diverse perspectives and viewpoints.
> 
> -- ChatGPT

**Comment by dessalines** [comment]
> I think [a trust level system like the one from discourse](https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/) would be more effective, that way all the responsibility doesn't fall on a few users that will get tired fast.

The stackoverflow style "power-user" / reputation-powered system, where power-users get extra abilities, is something **I've specifically wanted to avoid** with lemmy. Those systems encourage gamification and engagement solely for its own sake, and make networks hostile to new users / contributers. It encourages cliquishness and hostility towards anyone new, or who doesn't have a "high reputation score".

> I think it's time to concede that the moderation system isn't working. Many communities don't have moderation because the moderators haven't connected to the platform in months and people still don't use /c/community_requests.

That mods sometimes abandon communities, and that admins have to transfer ownership to new mods, does not mean that moderation is broken. 

To your suggestions, it seems like you haven't used lemmy yet. It already has has upvoting, reporting, open modlogs, human moderation, mod tools, and rule lists.

To your problems with lemmy moderation: bias, lack of training, lack of consistency... all of them affect human moderators however they are chosen.

**Comment by Nutomic** [comment]
> The stackoverflow style "power-user" / reputation-powered system, where power-users get extra abilities, is something I've specifically wanted to avoid with lemmy. Those systems encourage gamification and engagement solely for its own sake, and make networks hostile to new users / contributers. It encourages cliquishness and hostility towards anyone new, or who doesn't have a "high reputation score".

Isnt that more due to the points they award for different actions, and display similar to Reddit karma? I dont think the gamification problem would happen if trust levels are given manually by admins. I also think its okay to provide such gamification as an optional feature, so different instances can organize in the way they want. Importantly it should be optional and disabled by default.

> That mods sometimes abandon communities, and that admins have to transfer ownership to new mods, does not mean that moderation is broken.

Its true that abandoned communities could be handled better. Maybe they should be locked automatically, or flagged for admin review if the mods have been inactive for a certain amount of time.

**Comment by dessalines** [comment]
Yes, its due to the points they award, but even if "trust levels" were manually given (in the same way that admins appoint mods), you're giving extra abilities to certain users. Its those "extra abilities", that cause problems. 

- They might start out innocous enough, but it usually leads to certain users having abilities over others, and being able to domineer over them in certain ways.
- There's also the complication that comes from maintaining all kinds of different levels and abilities, and differing viewpoints as to what those abilities and levels should be. I'd pretty much guarantee that you can't get two people to agree on what set of extra abilities should be granted to what level of trust.

If its just *visual* gamification...we've *sort of* already come out against that, since we discourage karma-seeking by not displaying user's comment and post score totals. 

On the abandoned communities thing... I'd def be open to code-solutions to that problem, rather than just bugging admins in a request community. I'm not sure what that should be tho... maybe a "stale", and "stale_admins" column on community might help, and periodic queries or sql thingies that check for that.

**Comment by JediMaster25** [comment]
I understand your concerns about a power-user or reputation-based system, but I think it's important to consider the future of AI and its potential impact on moderation. As AI becomes more advanced and matches human intelligence, we can't just rely on asking a few questions to determine whether or not they should be allowed on the platform. A trust level system could potentially provide a more objective way to assess the trustworthiness and reliability of AI users. While it may not be perfect, it could be a step towards ensuring that our moderation system can adapt and evolve as AI technology advances. We need to start thinking about the future and how we can ensure that our moderation systems are prepared for the potential integration of AI.

**Comment by Nutomic** [comment]
> - They might start out innocous enough, but it usually leads to certain users having abilities over others, and being able to domineer over them in certain ways.

Thats already the case with mod and admin roles.

> - There's also the complication that comes from maintaining all kinds of different levels and abilities, and differing viewpoints as to what those abilities and levels should be. I'd pretty much guarantee that you can't get two people to agree on what set of extra abilities should be granted to what level of trust.

Also sounds similar to the differences in opinion regarding other Lemmy features. I dont think it would be so bad, nor is it a good reason to entirely leave out a useful feature.

 > If its just visual gamification...we've sort of already come out against that, since we discourage karma-seeking by not displaying user's comment and post score totals.

I agree about this, but surely there are also ways to implement trust levels without gamification. For example by awarding them manually.

**Comment by binary3141** [comment]
maybe just an "untrust levels"

**Comment by informapirata** [comment]
> 

I agree!

With our feddit.it instance we initially decided not to allow any user to create a community, but we created a community specifically for requests of this type, from users who are sufficiently motivated to manage a community. Later we decided to keep this system which has received strong approval from users and has not led to any feeling of centralization.

in some cases, users who did not feel confident enough to manage the moderation of a community themselves were accompanied by the instance administrators who act as additional moderators of the community, so that they could explain any new moderation features or, in cases more urgent, directly take charge of the content moderation.

the result is that less than a year after the opening of our instance, feddit.it has become one of the five most active instances and records a very low rate of dead communities.

If I think that much more established software equipped with advanced social features, such as Friendica, have not yet managed to adopt an advanced and practical content moderation and signaling system like the one achieved by lemmy, I would say that the level of satisfaction on our part it's really very high! And I take the opportunity to congratulate the developers for having created such a solid and functional toy...
